rfi_file_monitor/utils/__init__.py

```diff
@@ -103,14 +104,28 @@ def get_md5(fname: os.PathLike) -> str:
             hash_md5.update(chunk)
     return hash_md5.hexdigest()
 
+def get_file_creation_timestamp(file_path: os.PathLike):
+    # get creation time, or something similar...
+    # https://stackoverflow.com/a/39501288
+    if platform.system() == 'Windows':
```

Comment:
this is interesting. When did it change that getting a timestamp platform dependent?

Reply:
It's a bit of a mess indeed. This seems to be most reliable solution